test/functional/artefacts_controller_test.rb

```diff
@@ -152,6 +152,19 @@ class ArtefactsControllerTest < ActionController::TestCase
         get :new
         assert_select "li[class~=active] a[href=/artefacts/new]"
       end
+
+      should 'render the tags partial if publisher is not an untaggable app' do
+        Settings.expects(:untaggable_apps).returns(%w{ smartanswers testapp }).at_least(1)
```

Comment:
Also, when using the `%w` notation, can we stick to parens and remove the leading and trailing spaces per the style guide?